package.json

```diff
@@ -14,7 +14,7 @@
     "lint:ts": "tslint -c tslint.json -e './packages/**/node_modules/**/*.ts' './packages/**/*.ts'",
     "lint": "npm-run-all --parallel lint:*",
     "test:es": "npm run lerna-test",
-    "test:ts": "tsr packages/zipkin/test-types/*.test.ts --noAnnotate --libDeclarations && mocha --opts test/mocha-types.opts",
+    "test:ts": "tsr packages/*/test-types/*.test.ts --noAnnotate --libDeclarations && mocha --opts test/mocha-types.opts",
```

Comment:
Updated this to process `test-type/*.test.ts` tests in all packages. This reflects the second mocha command  which   uses `test/mocha-types.opts` and attempts to run test-type tests in all packages.